src/main/java/objects/character/Player.java

```diff
@@ -28,10 +32,11 @@ public class Player extends Character {
     private SkillType skillType;
     private int speed;
     private int money;
+    private final ArrayList<Skill> skillList;
```

Comment:
Probably does not need to be final, as later we might want to swap and change skills around!